### PR TITLE
feat(pipeline): add overview page to 3000 pipeline ui

### PR DESCRIPTION
**Pull request by thmsobrmlr**
## Problem

The 3000 UI for pipeline needs an overview page.

## Changes

This PR adds it.

## How did you test this code?

:eyes:

### CODE REVIEW

File: frontend/src/scenes/pipeline/Overview.tsx [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+                {metrics.totals} events processed with {percentage(failureRate)} failures (a total of {metrics.failures}{' '}
+                event(s)) in the last 7 days.
+            </span>
+        )
+    }
+
+    return <span>{metrics.totals} events processed without errors in the last 7 days.</span>
+}
+
+const StatusIndicator = ({ enabled, metrics }: StatusIndicatorProps): JSX.Element => {
```
Comment (tiina303) [inline frontend/src/scenes/pipeline/Overview.tsx:61]: Idea: let's remove the status indicator completely:
1. lets not show disabled apps at all (there could be lots of them and they aren't relevant in the moment)
2. the sparkLine gives us a great overview of how the app is doing already, having another colored dot could create more confusion than help 🤷‍♀️

File: frontend/src/scenes/pipeline/Overview.tsx [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+                    <div className="mr-1">
+                        <AppMetricSparkLine config={destination} />
+                    </div>
+                    <More overlay={<DestinationMoreOverlay destination={destination} />} />
+                </>
+            }
+            additionalInfo={
+                <div className="flex gap-1">
+                    {destination.backend === PipelineAppBackend.Plugin ? (
+                        <LemonTag type="primary">Plugin</LemonTag>
```
Comment (tiina303) [inline frontend/src/scenes/pipeline/Overview.tsx:229]: let's remove this - it's not relevant to the user, if we want to keep something I'd do frequency "realtime" vs "hourly" vs "daily" vs ...

**Review by tiina303 — COMMENTED** [review]
Looking great 🙌  just a couple of thoughts to switch/remove smaller details ... 

The app icons are quite nice to quickly visually find the app, maybe it could be in front of the name 🤷‍♀️  in half the size we have it in the Destination etc page (if we remove status indicator we have space for it?)

**Comment by thmsobrmlr** [comment]
Thanks for the feedback, these are exactly the comments I've been looking for. Have changes things along your suggestions and also removed the pause and delete buttons from the ...-menu, since the apps wouldn't be visible after disabling any more.

**Comment by tiina303** [comment]
This is good, we can ship it as is.

🤔  now looking at it again I'm wondering ... since we didn't go for the idea of showing it actually as a graph, would it be better for the user if we showed things exactly the same way as on the transformation & destination pages (filtering only for stuff that's enabled) - we seem to be using the same amount of space? It would make sense for these to be boxes if we actually had some sort of a graph.

**Review by Twixes — APPROVED** [review]
Agreed, let's get this in.

**Pull request merged**